app/client/templates/exhibitor/payment.js

```diff
@@ -0,0 +1,78 @@
+Template.payment.created = function () {
+  //
+};
+
+Template.payment.helpers({
+  //
+});
+
+Template.payment.rendered = function () {
+  $('select').material_select();
+
+  Session.set('paymentErrors', 0);
```

Comment:
Where is the check of this value to stop creating errors and tell user to email directly?